content/contributing-terms.en.md

```diff
@@ -0,0 +1,465 @@
+
+The types might not always match the exact name given by the service provider. For example, some providers might call “Terms and Conditions” or “Terms of Use” what some others call “Terms of Service”. The terms type does not have to match the exact name, it only has to match the _commitment_ that is taken.
+
+If the terms you want to add match no existing type, you can [suggest a new one](https://github.com/OpenTermsArchive/engine/discussions/categories/document-types).
+
+##### Defining new terms types
+
+Before defining a new terms type, please note that wanting to multiply terms types is usually a symptom that the service needs to be broken down into several services. For example, rather than considering that Facebook has several specific variations of “Terms of Service”, it is more accurate to declare “Terms of Service” for the services “Facebook” (social network service for the general public), “Facebook Ads” (ads service for advertisers) and “Facebook Payments” (payment service for developers). On the other hand, the “Google Ads” service is a commercial suite acting as an umbrella for several pieces of software that all share the same terms, and there is thus no need to separate each of them. See practical guidelines for [provider prefixing](declarations-guidelines.md#provider-prefixing).
+
+In order to guide usage and disambiguate synonyms, we characterise each terms type along three dimensions of the commitment that is being taken in it:
+
+1. The `writer` of the document;
+2. the targeted `audience`;
+3. the `object` of the commitment.
+
+A terms type thus looks like:
+
+```json
+{
+  …
+  "Privacy Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "personal data"
+    }
+  },
+  …
+}
+```
+
+Please note that we do not want [service-specific types](https://github.com/OpenTermsArchive/engine/pull/89) such as “Twitter Privacy Policy”. Terms types should be generic, even if only one service uses them at a given time. Otherwise, duplication occurs and [important efforts](https://github.com/OpenTermsArchive/engine/pull/88) have to be deployed to deduplicate. The triptych form “writer / audience / object” is precisely used to avoid this sort of duplication.
+
+### Testing your declaration
+
+You can test the declarations you created or changed by running the following command:
+
+```
+npm test [$service_id [$another_service_id …]]
+```
+
+Since this operation fetches documents and could be long, you can also validate the declaration structure only:
+
+```
+npm run test:schema [$service_id [$another_service_id …]]
+```
+
+#### Linting
+
+In order to ensure consistency across declarations, all declarations files have to be formatted homogeneously.
+
+In order to achieve this, you can use the following command:
+
+```
+npm run lint [$service_id [$another_service_id …]]
+```
+
+## Maintaining declarations
```

Comment:
When a title element follows a `pre` element there is some space missing

<img width="740" alt="Capture d’écran 2023-03-08 à 14 29 35" src="https://user-images.githubusercontent.com/1098708/223725662-3b5cd951-7a5a-4e49-90d2-1d4270c2d965.png">